docs/Docker-Container.md

```diff
@@ -0,0 +1,42 @@
+# Docker Container
+
+## Concept
+
+A Docker Container (or Docker Compose configuration) designed to test WordPress themes and plugins, especially under different combinations and/or configurations.
+
+### Examples
+
+This might be used to test a single WordPress plugin across multiple versions of WordPress + PHP:
+
+```yaml
+env:
+  - PHP_VERSION=7.4 WP_VERSION=5.3
+  - PHP_VERSION=7.0 WP_VERSION=4.9
+```
+
+It may also be used to test certain combinations of plugins:
+
+```yaml
+env:
+  - WP_PLUGINS=jetpack@latest,woocommerce@3.7
```

Comment:
This makes sense. It should also allow installing via zip. For example `caldera-forms@https://github.com/CalderaWP/Caldera-Forms/releases/download/1.8.3-beta.1/1.8.3-beta.1.zip` or `caldera-forms@gh-1.8.3-beta.1`.

wp-cli can install plugin by WordPress.org slug or ZIP file https://developer.wordpress.org/cli/commands/plugin/install/